### PR TITLE
Resolve NEXT_MAJOR comments

### PR DESCRIPTION
**Pull request by franmomu**
I've removed the country flags, ref: https://github.com/sonata-project/SonataTranslationBundle/pull/281

### CODE REVIEW

File: src/Checker/TranslatableChecker.php [hunk 1]
```diff
@@ -13,7 +13,6 @@
 
 namespace Sonata\TranslationBundle\Checker;
 
-use Sonata\TranslationBundle\Traits\Gedmo\PersonalTranslatable;
 use Sonata\TranslationBundle\Traits\Gedmo\PersonalTranslatableTrait;
 use Sonata\TranslationBundle\Traits\Translatable;
```
Comment (VincentLanglet) [inline src/Checker/TranslatableChecker.php:17]: Isn't this unused ?

Comment (franmomu) [reply src/Checker/TranslatableChecker.php]: yes, thanks

Comment (VincentLanglet) [reply src/Checker/TranslatableChecker.php]: Why PhpCsFixer didn't report this ?

**Review by VincentLanglet — APPROVED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thank you @franmomu

**Pull request merged**